File: Aerolinea/web/js/Asiento.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
function Asiento(numero,estado,numero_viaje){
	this.Asiento(numero,estado,numero_viaje);
}
Asiento.prototype={
        numero:"",
        estado:false,
        numero_viaje:0,
	Asiento:function(numero,estado,numero_viaje){
                this.numero=numero;
		this.estado=estado;
		this.numero_viaje=numero_viaje;
	},
        toString:function(){
	  return this.numero;
	}
}
Asiento.from= function(plain){
    asiento = new Asiento(plain.numero,plain.estado,plain.numero_viaje);
    return asiento;
};
Asiento.to= function(asiento){
    return {
        _class : 'Asiento',
        numero: asiento.numero,
        estado: asiento.estado,
        numero_viaje : asiento.numero_viaje
    };
};

File: Aerolinea/src/java/airline/model/Asiento.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package airline.model;
import java.io.Serializable;
/**
 *
 * @author AndreyCh
 */
public class Asiento implements Serializable, Jsonable{
    int codigo;
    String numero;
    boolean estado;
    Viaje viaje;

    public Asiento(int codigo, String numero, boolean estado, Viaje viaje) {
        this.codigo = codigo;
        this.numero = numero;
        this.estado = estado;
        this.viaje = viaje;
    }

    public Asiento(String numero, boolean estado, Viaje viaje) {
        this.numero = numero;
        this.estado = estado;
        this.viaje = viaje;
    }
    
    public Asiento() {
    }

    public int getCodigo() {
        return codigo;
    }

    public String getNumero() {
        return numero;
    }

    public boolean isEstado() {
        return estado;
    }

    public Viaje getViaje() {
        return viaje;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

    public void setEstado(boolean estado) {
        this.estado = estado;
    }

    public void setViaje(Viaje viaje) {
        this.viaje = viaje;
    }

    
}
